plugins/topology/README.md

```diff
@@ -128,6 +128,14 @@ The Topology plugin enables you to visualize the workloads such as Deployment, J
         app.openshift.io/vcs-ref: <GIT_REPO_BRANCH>
   ```
 
+- The following label is added to workload resources, such as Deployments to display runtime icon in the topology nodes:
+
+  ```yaml title="deployment.yaml"
+    ...
+    labels:
+      app.openshift.io/runtime: <RUNTIME_NAME>
+  ```
+
```

Comment:
@debsmita1 I think we should also mention if this label is not present then this label `app.kubernetes.io/name` is used to get the icon so that the user can use either of these two.